helmfile/charts/gatekeeper/podsecuritypolicies/values.yaml

```diff
@@ -0,0 +1,45 @@
+# Add pods that should override the restricted constraints
+# <namespace>:
+#   <service-name>:
+#     labelsSelector:
+#       foo: bar
+#     # Every "allow" is an override, and is optional.
+#     # If a override is NOT part of "allow", the restricted constraint is applied
+#     allow:
+#       allowedCapabilities:
+#         - NET_BIND_SERVICE
+#         - ...
+#       hostPort: <true/false>
```

Comment:
```suggestion
#       hostNetwork: <true/false>
```
Should we also include the range for `hostPorts`?

Reply:
One issue with gatekeepers HostNetworkPorts constraint is it only allows for one range. 
But we decided that it is okay to open up for 1-65535 as we have network policies.
But yeah maybe it is better to call it `hostNetwork`